SpriteSheet.hpp

```diff
@@ -9,7 +9,7 @@
 namespace spic {
 	class SpriteSheet {
 	public:
-		SpriteSheet(Vector2 frameSize, const std::string& url, int spriteCount);
+		SpriteSheet(Vector2<int> frameSize, const std::string& url, int spriteCount);
```

Comment:
Is de reden waarom deze Vector van type int is, omdat frameSize het aantal pixels aangeeft?

Reply:
Dat is exact de reden